src/vector/defn.rs

```diff
@@ -206,7 +203,7 @@ impl<'a> GeomField<'a> {
     /// Get the name of this field.
     pub fn name(&'a self) -> String {
         let rv = unsafe { gdal_sys::OGR_GFld_GetNameRef(self.c_field_defn) };
```

Comment:
`rv` is owned by `c_field_defn` so we don't need to free it?

Reply:
Yes, owned by `OGRGeomFieldDefn`.